files/reset_openstack_environment.yml

```diff
@@ -61,6 +61,11 @@
       # Ignore the case where the server does not exist.
       ignore_errors: true
 
+    # Stop the servers first for a more graceful shutdown.
```

Comment:
Why are you doing this?  Is this because of NVMe issues?  If so, that will not help, since you're only stopping ansible-host here, aren't you?  The rest of the VMs is deleted on stack delete.

Reply:
Yes it was for NVMe issues, and I realize that it will not help. I will pull this out.